.github/workflows/integration-tests.yml

```diff
@@ -39,12 +29,18 @@ jobs:
         run: |
           docker system prune -af
           docker volume prune -f
+      # Cleaning the runner is important to free enough space to build rancher, otherwise the build will fail
       - name: Clean runner
```

Comment:
Just a note that it feels that this could, eventually, turn into an action itself, as it's repeated multiple times.